### ISSUE TITLE
Wallet backup

### ISSUE THREAD

**Issue by codewiz** [manuelsc/Lunary-Ethereum-Wallet#16]
Is there no way to backup the (password encrypted) private key of the wallet?

If backups are already supported, then the feature is insufficiently discoverable. It is good practice for any cryptocurrency wallet to recommend users to backup the wallet, either at creation-time, or after  it receives funds for the first time.

**Comment by manuelsc** [comment]
Press and hold on the wallet you wish to export and select export. Same intuitive gesture as in every other app to bring up some options on a selected item.

**Comment by codewiz** [comment]
Thanks, now I found it.

I personally think it's hard to discover: I looked in the side menu and in the "+" menu in the wallet view, but did not even imagine there was a long-press contest menu in the wallet list.

Before filing this bug, I asked another Lunary user how he backed up his wallet, and he told me that he didn't do it and didn't know how to do it. Other wallet apps actively remind users to make backups and provide a secure workflow to do it (for instance, save an encrypted file in Google Drive, or send it to yourself by email).
